configs/restorers/dic/dic_gan_x8c48b6_g4_500k_CelebAHQ.py

```diff
@@ -0,0 +1,159 @@
+    pixel_loss=dict(type='L1Loss', loss_weight=1.0, reduction='mean'),
+    align_loss=dict(type='MSELoss', loss_weight=0.1, reduction='mean'),
+    feature_loss=dict(
+        type='FeatureLoss',
+        pretrained=pretrained_light_cnn,
+        loss_weight=0.1,
+        criterion='l1'),
+    gan_loss=dict(
+        type='GANLoss',
+        gan_type='vanilla',
+        loss_weight=0.005,
+        real_label_val=1.0,
+        fake_label_val=0))
+# model training and testing settings
+train_cfg = dict(fix_iter=10000, disc_steps=2)
+test_cfg = dict(metrics=['PSNR', 'SSIM'], crop_border=scale)
+
+# dataset settings
+train_dataset_type = 'SRFacialLandmarkDataset'
+val_dataset_type = 'SRFolderGTDataset'
+test_dataset_type = 'SRFolderGTDataset'
+train_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='gt',
+        flag='color',
+        channel_order='rgb',
+        backend='cv2'),
+    dict(
+        type='Resize',
+        scale=(128, 128),
+        keys=['gt'],
+        interpolation='bicubic',
+        backend='pillow'),
+    dict(
+        type='Resize',
+        scale=1 / 8,
+        keep_ratio=True,
+        keys=['gt'],
+        output_keys=['lq'],
+        interpolation='bicubic',
+        backend='pillow'),
+    dict(
+        type='GenerateHeatmap',
+        keypoint='landmark',
+        ori_size=256,
+        target_size=32,
+        sigma=1.),
+    dict(
+        type='Normalize',
+        keys=['lq', 'gt'],
+        mean=[129.795, 108.12, 96.39],
+        std=[255, 255, 255]),
+    dict(type='ImageToTensor', keys=['lq', 'gt', 'heatmap']),
+    dict(
+        type='Collect',
+        keys=['lq', 'gt', 'heatmap', 'landmark'],
+        meta_keys=['gt_path'])
+]
+valid_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='gt',
+        flag='color',
+        channel_order='rgb',
+        backend='cv2'),
+    dict(
+        type='Resize',
+        scale=(128, 128),
+        keys=['gt'],
+        interpolation='bicubic',
+        backend='pillow'),
+    dict(
+        type='Resize',
+        scale=1 / 8,
+        keep_ratio=True,
+        keys=['gt'],
+        output_keys=['lq'],
+        interpolation='bicubic',
+        backend='pillow'),
+    dict(
+        type='Normalize',
+        keys=['lq', 'gt'],
+        mean=[129.795, 108.12, 96.39],
+        std=[255, 255, 255]),
+    dict(type='ImageToTensor', keys=['lq', 'gt']),
+    dict(type='Collect', keys=['lq', 'gt'], meta_keys=['gt_path'])
+]
+test_pipeline = valid_pipeline
+
+data = dict(
+    workers_per_gpu=4,
+    train_dataloader=dict(samples_per_gpu=2, drop_last=True),
+    val_dataloader=dict(samples_per_gpu=1),
+    test_dataloader=dict(samples_per_gpu=1),
+    train=dict(
+        type='RepeatDataset',
+        times=60,
+        dataset=dict(
+            type=train_dataset_type,
+            gt_folder='data/CelebA-HQ/train_256/all_256/',
```

Comment:
See https://mmediting.readthedocs.io/en/latest/inpainting_datasets.html?highlight=CelebA-HQ#celeba-hq-dataset for the folder name for this dataset

Reply:
both `dic` and `dic-gan` should be changed.

Reply:
okay, pls do